app/src/main/java/org/myspecialway/common/Ext.kt

```diff
@@ -106,5 +107,13 @@ fun Button.enable(enable: Boolean) = when (enable) {
         this.isEnabled = false
         animate().setDuration(400).setInterpolator(AccelerateDecelerateInterpolator()).alpha(.5f).start()
     }
-
+}
+class Run {
+    companion object {
+        fun after(delay: Long, process: () -> Boolean) {
+            Handler().postDelayed({
```

Comment:
You should pass the Handler to the method instead of creating a new one.
1. It will make it testable.
2. If someone calls it from a thread that doesn't have a looper it will crash.
3. Less garbage as Handlers can be reused.